sov-modules/sov-modules-macros/src/dispatch/common.rs

```diff
@@ -108,9 +109,7 @@ impl<'a> StructDef<'a> {
         quote::quote! {
             // This is generated code (won't be exposed to the users) and we allow non camel case for enum variants.
             #[allow(non_camel_case_types)]
-            // TODO we should not hardcode the serialization format inside the macro:
-            // https://github.com/Sovereign-Labs/sovereign/issues/97
-            #[derive(borsh::BorshDeserialize, borsh::BorshSerialize, ::core::fmt::Debug, PartialEq)]
+            #[derive(borsh::BorshDeserialize, borsh::BorshSerialize, ::core::fmt::Debug, PartialEq, #(#serialization_attrs),*)]
```

Comment:
Now we can remove `derive(borsh::BorshDeserialize, borsh::BorshSerialize` because these `derive` will be passed via `serialization_attrs` right?